src/backend/api/meals.js

```diff
@@ -0,0 +1,268 @@
+
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//Week- 3
+
+//Returns all meals that are cheaper than maxPrice. Data type Number. api/meals?maxPrice=90
+
+router.get('/maxPrice', async (req, res) => {
+  try {
+    const maxPrice = req.query.maxPrice;
+
+    if (maxPrice !== undefined && isNaN(maxPrice)) {
```

Comment:
In JavaScript, it is always safer to check against both null and undefined. You can make use of the Falsy concept here and simply treat the value as a boolean: https://developer.mozilla.org/en-US/docs/Glossary/Falsy

The only thing to keep in mind is that 0 is also consider "false", so you may need to check for example like this: 
`if (maxPrice || maxPrice === 0)`